docs/sources/reference/commandline/cli.md

```diff
@@ -2181,6 +2181,19 @@ mount the volumes in read-only or read-write mode, respectively. By default,
 the volumes are mounted in the same mode (read write or read only) as
 the reference container.
 
+Labeling systems like SELinux require proper labels be placed on volume content
+mounted into a container, otherwise the security system might prevent the
+processes running inside the container from using the content. By default,
+volumes are not relabeled.
```

Comment:
How did the concept of "relabeling" enter into this?  Or do you mean you can only label a volume once and after that you can't relabel it?